dask/dataframe/io/parquet/core.py

```diff
@@ -796,6 +797,10 @@ def to_parquet(
         If not specified, files will created using the convention
         ``part.0.parquet``, ``part.1.parquet``, ``part.2.parquet``, ...
         and so on for each partition in the DataFrame.
+    filesystem: "fsspec", "arrow", fsspec.AbstractFileSystem, or pyarrow.fs.FileSystem
+        Filesystem backend to use. Note that the "fastparquet" engine only
+        supports "fsspec" or an explicit ``pyarrow.fs.FileSystem`` object.
```

Comment:
Ah - Looks like there is an error in the `read_parquet` doc-string that you are copying here. Pretty sure this should be:

```
supports "fsspec" or an explicit ``fsspec.AbstractFileSystem`` object.
```

Reply:
Fixed!